etc/picongpu/taurus-tud/V100_picongpu.profile.example

```diff
@@ -1,76 +1,83 @@
+## `picongpu.profile` for the `ml` partition on taurus@TU-Dresden
+## The `ml` partition is equipped with Nvidia V100 GPUs
+## Nodes in the `ml` partition are Power9 systems which is different
+## from the headnodes. Therefore, compile on a node!
+
 # Name and Path of this Script ############################### (DO NOT change!)
 export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
 
-# User Information ######################################### (edit those lines)
+# User Information ######################################### (edit these lines)
 #   - automatically add your name and contact to output file meta data
 #   - send me a mail on batch system jobs: BEGIN, END, FAIL, REQUEUE, ALL,
 #     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
 export MY_MAILNOTIFY="ALL"
```

Comment:
please set the default notify to `NONE` see https://github.com/ComputationalRadiationPhysics/picongpu/pull/2743

Could you also update the comment above to https://github.com/ComputationalRadiationPhysics/picongpu/pull/2743/files#diff-1a92a2a50aa463b36ac1ba265a8b0524R6 because the option `NONE` is missing in the description